### ISSUE TITLE
Add Support to Retrieve the Pull Request's Quality Status

### ISSUE THREAD

**Issue by xuanan93** [phwt/sonarqube-quality-gate-action#25]
Please update get PR, currently I see your project only get status of branch.
/dashboard?id=repo&branch=develop
I am using PR so I can't get report, can you add get PR, the URL will be like this
dashboard?id=repo&pullRequest=69.

Thanks,

**Comment by phwt** [comment]
Hi @xuanan93! You can use `branch` input to specify the branch that this action will get the quality status from, you can configure the action similar to the below:

```diff
jobs:
  sonarqube:
    runs-on: ubuntu-latest
    steps:
      - uses: phwt/sonarqube-quality-gate-action@v1
        id: quality-gate-check
        with:
          sonar-project-key: ${{ secrets.SONAR_PROJECT_KEY }}
          sonar-host-url: ${{ secrets.SONAR_HOST_URL }}
          sonar-token: ${{ secrets.SONAR_TOKEN }}
          github-token: ${{ secrets.GITHUB_TOKEN }}
+         branch: main
```

In the background, this action will call the [`GET /api/qualitygates/project_status`](https://next.sonarqube.com/sonarqube/web_api/api/qualitygates/project_status) endpoint to retrieve the quality gate status, if the `branch` input is specified, the action will append the `branch` parameter to this endpoint, similar to the below:

```
GET /api/qualitygates/project_status?projectKey=your-project-key&branch=your-branch
```

For reference, the request to SonarQube API is handled in [`src/modules/sonarqube-api.ts`](https://github.com/phwt/sonarqube-quality-gate-action/blob/e6daa1b68c5868fa96f7a3a21b987da19585d0ab/src/modules/sonarqube-api.ts) file.

**Comment by phwt** [comment]
Hi again @xuanan93, I was sleepy and read your issue wrong, the support for pull request will be released with #30
